### PR TITLE
Full Mesh Traffic Sanity verification 

### PR DESCRIPTION
**Pull request by sreejithsreekumaran**
### Description of PR

Summary:
Fixes # (issue)

### Type of change

- [ ] Bug fix
- [ ] Testbed and Framework(new/improvement)
- [x] Test case(new/improvement)

### Back port request
- [ ] 202012
- [ ] 202205
- [ ] 202305
- [ ] 202311
- [ ] 202405

#### How did you do it?

ptf64 topology is used to have 64 ports of the DUT connected
to 64 PTF ports.

#### How did you verify/test it?
Ran the test on a T1 ptf-64 topology

#### Any platform specific information?

This is a Cisco specific test case

### CODE REVIEW

**Comment by kevinskwang** [comment]
@sreejithsreekumaran  please fill all the sections of the commit message

File: tests/qos/qos_sai_base.py [hunk 1]
```diff
@@ -132,13 +132,17 @@ def runPtfTest(self, ptfhost, testCase='', testParams={}, relax=False, pdb=False
         """
         custom_options = " --disable-ipv6 --disable-vxlan --disable-geneve" \
                          " --disable-erspan --disable-mpls --disable-nvgre"
+        # Append a suffix to the logfile name if log_suffix is present in testParams
+        log_suffix = testParams.get("log_suffix", "")
+        logfile_suffix = "_{0}".format(log_suffix) if log_suffix else ""
```
Comment (kevinskwang) [inline tests/qos/qos_sai_base.py:137]: Please create a separate PR for this change

Comment (sreejithsreekumaran) [reply tests/qos/qos_sai_base.py]: I have removed it and will raise a separate PR.

**Comment by sreejithsreekumaran** [comment]
> @sreejithsreekumaran please fill all the sections of the commit message

Please check. I have updated now

**Comment by sreejithsreekumaran** [comment]
New PR https://github.com/sonic-net/sonic-mgmt/pull/14885 for logfile suffix.

File: tests/saitests/py3/sai_qos_tests.py [hunk 2]
```diff
@@ -311,10 +312,10 @@ def get_ip_addr():
 def get_tcp_port():
     val = 1234
     while True:
-        if val == 65535:
```
[the remark this answers is not in the document]
Comment (sreejithsreekumaran) [reply tests/saitests/py3/sai_qos_tests.py:314]: @kevinskwang Strange, I got email notification only today for this comment.

- The modulo operation ((val + 10) % 65535) will always yield values between 0 and 65534. Hence, val will never actually reach 65535.

- The updated logic where the ports reset to 1234 after reaching 65534 is for our requirement of needing an infinite supply of ports. This approach effectively reuses port numbers in a cyclic manner, ensuring that we always have ports available without hitting a hard limit, which is crucial in scenarios like full-mesh traffic sanity testing.

[the remark this answers is not in the document]
Comment (sreejithsreekumaran) [reply tests/saitests/py3/sai_qos_tests.py:314]: @kevinskwang please confirm

**Review by kevinskwang — APPROVED** [review]

**Pull request merged**